taqueria-vscode-extension/CHANGELOG.md

```diff
@@ -1,9 +1,66 @@
 # Change Log
 
-All notable changes to the "taqueria" extension will be documented in this file.
+## [Unreleased]
 
-Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
+## In Development
+
+These features will be launching in upcoming releases:
+- Jest plugin
```

Comment:
Are we calling it out as a Jest or a testing plugin?

Reply:
I'm not sure. I thought it was specific to Jest but maybe not...